java/configuration/src/main/java/sleeper/configuration/properties/instance/CommonProperty.java

```diff
@@ -194,36 +194,35 @@ public interface CommonProperty {
             .validationPredicate(Utils::isTrueOrFalse)
             .propertyGroup(InstancePropertyGroup.COMMON)
             .runCdkDeployWhenChanged(true).build();
-    UserDefinedInstanceProperty TABLE_INDEX_DYNAMO_POINT_IN_TIME_RECOVERY = Index.propertyBuilder("sleeper.tables.index.dynamo.pointintimerecovery")
+    UserDefinedInstanceProperty TABLE_INDEX_DYNAMO_POINT_IN_TIME_RECOVERY = Index.propertyBuilder("sleeper.common.table.index.dynamo.pointintimerecovery")
```

Comment:
We could just leave the table index properties starting with `sleeper.tables.index`, since this is plural?